userguide/storage.rst

```diff
@@ -1765,8 +1765,10 @@ device name, serial number, size, advanced power
 management settings, acoustic level settings, and whether
 :ref:`S.M.A.R.T.` tests are enabled. The pool associated with the disk
 is displayed in the :guilabel:`Pool` column. *Unused* is displayed if
-the disk is not being used in a pool. Click :guilabel:`COLUMNS` to
-adjust the table.
+the disk is not being used in a pool. Click :guilabel:`COLUMNS` and
```

Comment:
I don't know that we need to keep saying this.  If we do (and let's decide that), it can be a snippet.  But I think it should be mentioned once and then ignored.  Maybe a "how to use the UI" section at the beginning of the guide.